Tuprules.tup

```diff
@@ -46,7 +46,7 @@ export PKG_CONFIG_PATH
 !ld = |> ^ LINK %o^ $(CC) %f -o %o $(LDFLAGS) $(LDFLAGS_%o) |>
 !ar = |> ^ AR %o^ ar crs %o %f |>
 !dot = |> ^ DOT %f^ dot -Efontname="Vernada, serif" -Nfontname="Vernada, serif" -Efontsize=10 -Nfontsize=10 -Tpng %f > %o |>
-!cp = |> ^ CP %f -> %o^ cp %f %o |>
+!cp = |> ^ CP %f -> %o^ cp -- %f %o |>
```

Comment:
What did you need this for? There still aren't any files with dashes, as far as I can see.